Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -302,6 +302,15 @@
         "ApiCheck": "Windows.UI.Xaml.Controls.ColorPicker",
         "BadgeUpdateVersionRequired": "Fall Creators Update required",
         "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/controls/InfiniteCanvas.md"
+      },
+      {
+        "Name": "ImageCropper",
+        "Type": "ImageCropperPage",
+        "About": "ImageCropper control allows user to crop image freely.",
+        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper",
+        "XamlCodeFile": "ImageCropperXaml.bind",
+        "Icon": "/SamplePages/ImageCropper/ImageCropper.png",
```

Comment:
Can you add a codefile to show an example of the C# code from the ImageCropperPage code-behind to display in the sample app?

Reply:
I've added it.